go/internal/services/keys/get_migrated.go

```
@@ -0,0 +1,138 @@
+	// The section below is expected to populate this and we can use it to look up a key in the db
+	var h string
+
+	switch migration.Algorithm {
+	case db.KeyMigrationsAlgorithmGithubcomSeamapiPrefixedApiKey:
+		{
+			parts := strings.Split(rawKey, "_")
+			err = assert.Equal(len(parts), 3, "Expected prefixed api keys to have 3 segments")
+			if err != nil {
+				return nil, emptyLog, fault.Wrap(
+					err,
+					fault.Code(codes.URN(codes.Auth.Authentication.Malformed.URN())),
```

golangci-lint (2.2.2)

unnecessary conversion

(unconvert)